### PR TITLE
Migrate to new maps and traffic controls

### PR DESCRIPTION
**Pull request by zhangk1551**
Changes corresponding to:
https://github.com/inverted-ai/torchdrivesim/pull/44

### CODE REVIEW

**Comment by zhangk1551** [comment]
test video: https://drive.google.com/file/d/14BoYys4fY-tXQ2Ah2fY6Pc0OdZ0rHZtL/view?usp=sharing

File: torchdriveenv/gym_env.py
Comment (adscib) [inline torchdriveenv/gym_env.py]: Maybe include `carla_` in the location name? That will make it easier to incorporate non-carla maps in the future. Up to you.

File: torchdriveenv/gym_env.py
Comment (adscib) [inline torchdriveenv/gym_env.py]: Can you get it as `map_cfg.iai_location_name`?

**Review by adscib — COMMENTED** [review]
Looks good. I'll approve after the corresponding TDS PR is merged.

**Comment by zhangk1551** [comment]
Just found the video in this PR is also drive on the left. Let me check it.

**Review by adscib — APPROVED** [review]

**Pull request merged**
